likes/api.py

```diff
@@ -0,0 +1,108 @@
+from rest_framework.generics import get_object_or_404
+from author.models import Author
+from comment.models import Comment
+from likes.models import Like
+from post.models import Post
+from rest_framework import viewsets, status
+from rest_framework.response import Response 
+from knox.auth import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+from .serializers import LikeSerializer
+from author.serializer import AuthorSerializer
+
+import uuid
+import ast
+
+class PostLikeViewSet(viewsets.ModelViewSet):
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+    serializer_class = LikeSerializer
+
+    def get_post_likes(self, request, author_id, post_id):
+        query_set = Like.objects.filter(post=Post.objects.get(id=post_id))
+        response = LikeSerializer(query_set, many=True).data
+        
+        for likeObj in response:
+            likeObj['@context'] = "https://www.w3.org/ns/activitystreams"
```

Comment:
This should be get from request data, not hard coded. Though we probably don't have time to fix it now

Reply:
I didn't know what context is here so just hardcoded it